assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -15,9 +15,34 @@
  */
 package org.sakaiproject.assignment.impl;
 
-import static org.sakaiproject.assignment.api.AssignmentConstants.*;
-import static org.sakaiproject.assignment.api.AssignmentServiceConstants.*;
-import static org.sakaiproject.assignment.api.model.Assignment.Access.*;
+import static org.sakaiproject.assignment.api.AssignmentConstants.GRADEBOOK_INTEGRATION_ADD;
```

Comment:
The entire class AssignmentConstants is meant to be imported so it's the one correct use of using import *

Reply:
OK